fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/RelOpValue.java

```diff
@@ -894,6 +929,119 @@ private enum BinaryPhysicalOperator {
         GT_IDN(Comparisons.Type.GREATER_THAN, Type.TypeCode.UUID, Type.TypeCode.NULL, (l, r) -> null),
         GTE_NID(Comparisons.Type.GREATER_THAN_OR_EQUALS, Type.TypeCode.NULL, Type.TypeCode.UUID, (l, r) -> null),
         GTE_IDN(Comparisons.Type.GREATER_THAN_OR_EQUALS, Type.TypeCode.UUID, Type.TypeCode.NULL, (l, r) -> null),
+
+        IS_DISTINCT_FROM_BU(Comparisons.Type.IS_DISTINCT_FROM, Type.TypeCode.BOOLEAN, Type.TypeCode.NULL, (l, r) -> Objects.nonNull(l)),
```

Comment:
Can you adjust the naming for these? 
I think you're using the letter `U` from when those were unknown, but looking at the rest of the file, you'd want to use `N` for nulls.
For example `IS_DISTINCT_FROM_BU` -> `IS_DISTINCT_FROM_BN`